src/app/ngrx/reducers/episode.reducer.ts

```diff
@@ -1,10 +1,12 @@
-import { Action } from '@ngrx/store';
-import ActionTypes from '../actions/action.types';
+import { ActionTypes, ActionWithPayload, CmsEpisodeGuidPayload } from '../actions';
 import { EpisodeModel } from '../model';
 
 const initialState = [];
 
-export function EpisodeReducer(state: EpisodeModel[] = initialState, action: Action) {
+// TODO: should be able to use ActionWithPayload<All> here with the union type,
```

Comment:
I am quite dissatisfied not getting this All union type to work. None of the reducers are currently processing more than one action, but that's the whole point, that they can and might need to independent of what other reducers are using those actions for. If I want to use `ActionWithPayload<All>` here, I also need to wrap each case in an `if (action instanceof CmsEpisodeGuidAction)`, but the docs say it should "just work."